data.js

```diff
@@ -964,6 +964,14 @@ module.exports = [
     url: "https://github.com/organic-scholar/http.js",
     source: "https://raw.githubusercontent.com/organic-scholar/http.js/master/dist/http.js"
   },
+  {
+    name: "pico-ajax",
+    github: "udivankin/pico-ajax",
+    tags: ["fetch", "promise", "json", "xhr", "ajax", "rest", "http", "axios", "https"],
+    description: "Universal (browser and NodeJS), very tiny (browser version is ~1kb uncompressed) yet fully functional AJAX library with zero dependencies. Gives control over body and headers.",
+    url: "https://github.com/udivankin/pico-ajax",
+    source: "https://raw.githubusercontent.com/udivankin/pico-ajax/master/dist/browser/picoajax.min.js"
```

Comment:
Please change this to provide a human-readable source file.

From the microjs readme:

> The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified